book_source/06_reference/10_editing_this_book.Rmd

```diff
@@ -0,0 +1,62 @@
+# Editing this book {#bookediting}
+
+The file organization of this documentation can be described simply as follows:
+
+- Each **section** is in its own directory.
+- Each **chapter** is in its own file (within the corresponding section).
+
+Sections and chapters are rendered (and numbered) in alpha-numerical order of their corresponding file names.
+Therefore, each section directory and chapter file name should be **prefixed with a two-digit (zero-padded) number**.
+File and directory names should be as similar as possible to the name of the corresponding chapter or section.
+For instance, the file name for this chapter's source file is `01_introduction/03_how_to_use_book.Rmd`.
+This numbering means that if you need to create an additional chapter _before_ an existing one, you will have to renumber all chapters following it.
+You can save some typing by using following bash/zsh expansion like the following to augment a file number (example to augment this file from 3 to 4):
+
+```sh
+mv 01_introduction/{03,04}_how_to_use_book.Rmd
+```
```

Comment:
A useful tip, but I'd cut it to avoid overwhelming readers

Reply:
Fair enough. Removed in https://github.com/PecanProject/pecan/pull/2229/commits/36db4bf4f170e9f01053c9b225bd7a8b870bdd2c.